man/zetta_colors.5.md

```diff
@@ -1,20 +1,20 @@
-% exa_colors(5) v0.9.0
+% zetta_colors(5) v0.9.0
 
-<!-- This is the exa_colors(5) man page, written in Markdown. -->
+<!-- This is the zetta_colors(5) man page, written in Markdown. -->
 <!-- To generate the roff version, run `just man`, -->
 <!-- and the man page will appear in the ‘target’ directory. -->
 
 
 NAME
 ====
 
-exa_colors — customising the file and UI colours of exa
+zetta_colors — customising the file and UI colours of exa
 
 
 SYNOPSIS
 ========
 
-The `EXA_COLORS` environment variable can be used to customise the colours that `exa` uses to highlight file names, file metadata, and parts of the UI.
+The `ZETTA_COLORS` environment variable can be used to customise the colours that `exa` uses to highlight file names, file metadata, and parts of the UI.
```

Comment:
I'd say we should try and keep up compatibility as much as possible.
I therefore suggest to (at least) keep support for `EXA_COLORS`.
I'm open to adding `ZETTA_COLORS` but I'm not sure if it is wise to have two identical env vars for the same thing.

Reply:
I agree with keeping it (and other env vars) for backwards compatibility, I'm only unsure if it would be confusing to future users.

Renaming the env vars could potentially be its own issue and/or PR (as that would also require additional changes to the code).

Reply:
Actually, thinking about it just now, it is also kind of neat to have both exa and zetta using the same environment variables, so we probably should let them be for now.